lib/topics/index.js

```diff
@@ -90,11 +90,10 @@ Topics.prototype.getTopicPromises = function getTopicPromises(doneTopicPromise,
  * @param {string} topicName   - The name of the topic to generate
  * @returns {Promise} - A promise for the topic.
  */
-Topics.prototype.getErrorAndDoneTopicPromises = function getErrorAndDoneTopicPromises(config, topicName) {
+Topics.prototype.getErrorAndDoneTopicPromises = function getErrorAndDoneTopicPromises(topicName, config, payload) {
   //Creating a unique channel to get the response
-  // TODO move done/error prefixes here.
 
-  var topicUid = shortid.generate();
+  var topicUid = payload.topicUid || shortid.generate();
```

Comment:
This would never get any effect as in line 117 we already do: 
`payload.topicUid = payload.topicUid || shortid.generate();`